src/components/PageSetupDialog.vue

```diff
@@ -235,7 +235,13 @@
               v-model="form.dropCapDefaultFontSize"
             />
           </div>
-
+          <div class="form-group">
+            <label class="drop-caps-label">Line height</label>
```

Comment:
```suggestion
            <label class="drop-caps-label">Line Height</label>
```

For UI consistency, Height should be capitalized.